packages/core/src/api/nodeConversions/nodeConversions.ts

```diff
@@ -160,26 +187,22 @@ export function tableContentToNodes(
   return rowNodes;
 }
 
-/**
- * Converts a BlockNote block to a TipTap node.
- */
-export function blockToNode<BSchema extends BlockSchema>(
-  block: PartialBlock<BSchema>,
-  schema: Schema
+function blockOrInlineContentToContentNode(
```

Comment:
Why does this function also accept inline content? AFAIK it's only used once and it's with a block

Reply:
we also call it in `blockToNode`